notebooks/official/prediction/llm_streaming_prediction.ipynb

```diff
@@ -0,0 +1,791 @@
+{
```

Comment:
<p>It'd be better to use the official names of the models. See https://cloud.google.com/vertex-ai/docs/generative-ai/learn/models</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2255/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>